lib/admin/config.js

```diff
@@ -116,6 +116,12 @@ function fetchConfigGroup (code) {
 
       const selectedCryptos = _.defaultTo([], getCryptos(config, machineList))
 
+      if (devMode) {
+        const allowed = ['mock-ticker', 'mock-wallet', 'mock-exchange', 'mock-sms', 'mock-id-verify', 'mock-zero-conf']
+        const isValid = o => _.includes(o.code, allowed)
+        data.accounts = _.filter(isValid, data.accounts)
+      }
+
```

Comment:
I think there was a misunderstanding with the spec:

> l-a-s should only supply the mock-ticker, mock-wallet, mock-exchange, mock-sms, mock-id-verify, and mock-zero-conf options to l-a when ``--dev`` flag is present

Reading it over, I can see how it could be confusing. What I meant was that in devMode all options should be shown, but in non-devMode, all options besides these options should be shown. We want to hide this options in production mode because they are dangerous.

Let's do some unit tests for this to make the behavior easier to see.

Reply:
How would you suggest to unit test this?

Reply:
Maybe just create a separate function that take data.account and devMode as arguments and unit test that.